tests/functional/conftest.py

```diff
@@ -25,6 +29,14 @@
 )
 
 
+@pytest.fixture
+def environ():
+    environ = dict(os.environ)
```

Comment:
Fixture to make easier to access the os.environ version that contains all TEST_ENVIROMENT values.